frontend/src/components/LoginForm.tsx

```diff
@@ -62,6 +64,7 @@ const LoginForm = (props: any) => {
 
     if (isAuthenticated) {
       clearErrors();
+      clearModals();
```

Comment:
Why does this happen in the form too, not just in the modals themselves?

Reply:
because the form exists in the modal